java/src/test/java/scenario/ScenarioSteps.java

```diff
@@ -297,6 +310,34 @@ public ScenarioSteps() throws IOException {
         Then("the error message should contain {string}",
                 (String expected) -> assertThat(transactionInvocation.getError().getMessage()).contains(expected));
 
+        When("I stop the peer named {}", (String peer) -> {
+            exec("docker", "stop", peer);
+            runningPeers.put(peer, false);
+        });
+
+        When("I start the peer named {}", (String peer) -> {
+            exec("docker", "start", peer);
+            runningPeers.put(peer, true);
+            Thread.sleep(20000);
+        });
+    }
+
+    private static void startAllPeers() throws InterruptedException {
+        runningPeers.forEach((peer, running) -> {
+            if (!running) {
+                try {
+                    exec("docker", "start", peer);
+                } catch (IOException e) {
+                    e.printStackTrace();
+                    return;
```

Comment:
If this fails the tests are broken so I think we'd be better throwing an Exception and failing fast instead of continuing on for the tests to behave unexpectedly later